config/locales/en.yml

```diff
@@ -153,6 +153,7 @@ en:
     appeal_to_crown_court: Appeal to crown court
     appeal_to_crown_court_with_changes: Appeal to crown court with changes in financial circumstances
     class: Class
+    class_not_determined: "CLASS\xA0NOT\xA0DETERMINED"
```

Comment:
Is there no CSS way of preventing this badge from wrapping? I imagine there's no accessibility penalty by hard coding no-wrap chars like this?